AWSSDKSwiftCLI/Tests/AWSSDKSwiftCLITests/Models/ReleaseNotesBuilderTests.swift

```diff
@@ -209,10 +209,8 @@ class ReleaseNotesBuilderTests: XCTestCase {
             repoType: .awsSdkSwift,
             commits: testCommits,
             // Parametrize behavior of FeaturesReader with paths used to create JSON test files
-            featuresReader: FeaturesReader(
-                requestFilePath: "build-request.json",
-                mappingFilePath: "feature-service-id.json"
-            )
+            features: FeaturesReader().getFeaturesFromFile(),
+            featuresIDToServiceName: FeaturesReader().getFeaturesIDToServiceNameDictFromFile()
```

Comment:
During tests, the FeaturesReader just reads the feature & map files out of the temp project root created for testing.